### DISCUSSION TITLE
Support for .NET Framework removed?

### DISCUSSION THREAD

**Discussion by ghost** [mono/CppSharp#1789] [Q&A] [answered]
I tried to install both ClangSharp and CppSharp using nuget. It seems both only support .NET and not .NET Framework. Since I stuck with .NET Framework, I think I will give SharpGenTools a try. I don't want to upgrade yet.

**Comment by tritao** [comment, marked as answer]
You can still generate code that works with .NET Framework, but I don't think it's unreasonable to require a modern .NET for the tool itself.
